package.json

```diff
@@ -46,22 +50,18 @@
   "devDependencies": {
     "chai": "^3.5.0",
     "chai-resemble": "^0.4.1",
-    "grunt": "^0.4.5",
-    "grunt-eslint": "^20.0.0",
-    "grunt-mocha-cov": "^0.4.0",
-    "load-grunt-tasks": "^3.5.0",
-    "time-grunt": "^1.3.0"
+    "coveralls": "^2.13.1",
+    "eslint": "^4.2.0",
+    "istanbul": "^0.4.5",
+    "mocha": "^3.4.2",
+    "nyc": "^11.0.3"
   },
   "engines": {
     "node": ">=4.0"
   },
-  "config": {
-    "blanket": {
-      "pattern": [
-        "src/lib.js",
-        "src/uncss.js",
-        "src/utility.js"
-      ]
-    }
+  "nyc": {
+    "include": [
+      "src/**.js"
```

Comment:
Correct glob pattern for recursive search is `src/**/*.js`. Not an issue now, but will be if we add a subfolder in `src/` sometime.